src/Resources/views/admin_order/item_complaint.html.twig

```diff
@@ -5,21 +5,21 @@
     <input type="hidden" name="csrfToken" value="{{ pimcore_csrf.getCsrfToken(app.request.session) }}">
     <div class="modal-header">
         <h5 class="modal-title">{{ orderItem.getProductName() }} <small>{{ ('bundle_ecommerce.back-office.order.item-complaint')|trans([],'admin') }}</small></h5>
-        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+        <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close">
             <span aria-hidden="true">&times;</span>
         </button>
```

Comment:
<img width="453" alt="image" src="https://user-images.githubusercontent.com/11687066/231188815-b8812849-a64a-4d80-9a7e-94ed839878cc.png">
Otherwise we have the `x` twice.

```suggestion
        <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close">
        </button>
```